posts/reasons-to-build-our-application-using-angular.md

```diff
@@ -0,0 +1,116 @@
+---
+authors:
+- Anne Rosary 
+tags:
+- Angular
+date: 2022-10-17T12:21:50.000Z
+title: "Top Reasons to Build Your Next Application Using Angular"
+image: 
+---
+![angular](/images/2022/10/angular.png)
+
+While I was researching, why I need to learn Angular, I found out the following answers. I thought of sharing with you all!!
+Lets start to look why we need to have Angular in our next application.
+
+<h3>Angular is a full-fledged framework :</h3>
+
+   Out of the box, Angular has everything we need to build a business-level web application without the need for additional libraries.
+
+<h3>Modularity :</h3>
+
+Angular implements MVC by asking you to split your app into MVC components, then just let Angular do the rest. Angular manages your components for you and also serves as the pipeline that connects them.
+
+<h3>Data Binding :</h3>
+
+In Angular, this is the coolest concept with two-way binding and we would like to introduce it as an eye candy feature. Using NgModel you can change the data in both way template to class or class to template.
+
+
+![two-way-binding](/images/2022/10/two-way-binding.png)
+
+
+<h3>PWA and SPA :</h3>
+
+Angular Progressive Web Application (PWA) is a cost-friendly solution that allows websites to act like mobile apps. It reduces dependency on the network which significantly improves the user experience of the website.
+
+Caching in PWA works capably and conserves bandwidth whenever possible. This minimizes the risks of serving outdated content. 
+
+Angular also facilitates the development of Single Page Applications (SPA) which provides server side rendering capabilities that boost SEO(Search engine optimization) rankings. A single page application is a website or web application that dynamically rewrites a current web page with new data from the web server, instead of the default method of a web browser loading entire new pages. It also helps load the first page quickly and improve website performance on mobile and low-powered devices.
```

Comment:
```suggestion
Angular also facilitates the development of Single Page Applications (SPA) which provides server side rendering capabilities that boost SEO(Search engine optimization) rankings. A single page application is a website or web application that dynamically rewrites a current web page with new data from the web server, instead of the default method of a web browser loading entire new pages. It also helps load the first page quickly and improves website performance on mobile and low-powered devices.
```

Reply:
Changed it